### PR TITLE
Replacing header and footer navigation

### PR DESCRIPTION
**Pull request by znegrin**
This PR is related to the issue: https://github.com/src-d/backlog/issues/1404

I replaced the header and footer navigation to fit the new landing page.

image

image

Signed-off-by: Zuri Negrín <zurinegrin@gmail.com>

### CODE REVIEW

**Comment by vcoisne** [comment]
Content LGTM

File: hugo/layouts/partials/header.html [hunk 1]
```diff
@@ -12,10 +12,32 @@
             <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
                 <ul class="navbar-nav">
                     <li class="nav-item">
-                        <a class="nav-link" href="engine">source{d} Engine</a>
+                        <a class="nav-link" href="why-sourced">Why source{d}</a>
```
Comment (dpordomingo) [inline hugo/layouts/partials/header.html]: The `Why source{d}` and `Features` links have not visible hover state.

Normal state :point_down:
image

`Features` and `Resources` where hovered; `Features` almost didn't change :point_down:
image

File: hugo/layouts/partials/header.html [hunk 2]
```diff
@@ -12,10 +12,32 @@
...
                     <li class="nav-item">
-                        <a class="nav-link" href="enterprise">Enterprise</a>
+                        <a class="nav-link" href="features">Features</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="enterprise" id="navbarDropdownMenuLinkEnt" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Enterprise
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLinkEnt">
+                            <a class="dropdown-item" href="enterprise#solutions">Application Portfolio Management</a>
```
Comment (dpordomingo) [inline hugo/layouts/partials/header.html]: All four links inside `Enterprise` menu points to the same section.
Is it correct?

File: hugo/layouts/partials/footer.html [hunk 3]
```diff
@@ -8,14 +8,30 @@
...
+                    <li><a href="get-started">source{d} for Developers</a></li>
+                    <li><a href="get-started">source{d} for Managers</a></li>
+                    <li><a href="get-started">source{d} for Executives</a></li>
+                </ul>
+            </div>
+            <div class="col-sm pr-3">
+                <ul>
+                    <li><a class="footer-main" href="enterprise">Solutions</a></li>
+                    <li><a href="enterprise">source{d} Enterprise</a></li>
+                    <li><a href="enterprise#solutions">Application Portfolio Management</a></li>
```
Comment (dpordomingo) [inline hugo/layouts/partials/footer.html]: All four links inside `Enterprise` menu points to the same section.
Is it correct?

**Comment by dpordomingo** [comment]
Ready at staging:
https://landing-staging.srcd.run

**Comment by znegrin** [comment]
Fixed the :hover color on links and added anchors to the enterprise section so it points to the small sub-sections.

**Comment by dpordomingo** [comment]
Hi; Is there any new commit here? I found no new one.

**Comment by znegrin** [comment]
@vcoisne I just pushed the requested changes. I am just missing the Newsletter one as I wasn't sure of where to point it. Please let me know so I can add it properly. Thank you!

**Comment by znegrin** [comment]
I fixed the get-started links to point them to the individual tab anchors

**Comment by vcoisne** [comment]
Thank you @znegrin. 

Please make the following changes to the header:

- Add "Products" and "solutions" to header Nav (Why sourced, Products, Solutions, Community, Ressources, Company) 
- Solutions linking directly to the 4 new solution pages (See content in mocqups)
- Rename Features page - source{d} Community Edition and move it under "Products"
- Move source{d} Enterprise under Products as well
- List “newsletter” under “resources” 

For the footer: 

- Rename Features page - source{d} Community Edition and move it under "Products"
- Move source{d} Enterprise under Products as well

**Comment by znegrin** [comment]
Hi @vcoisne,

I was trying to find the new solution pages but I didn't find any mockup related to them. Can you please let me know where can I find them so I can design those. Once the pages are ready I will replace the navigation as if I do it now those links won't have a destination yet.

Thank you!

**Comment by znegrin** [comment]
Hi @vcoisne,

As requested on #409 I changed the IT Modernization links on header and footer.

image

image

**Comment by vcoisne** [comment]
@znegrin the solutions page are still work in progress for now. I should have them ready by tomorrow.

**Comment by dpordomingo** [comment]
If I understood properly @vcoisne requirements, it would be better if we perform the possible changes asap, and leave the new sections being developed for new PR, so the merged menu could be:


- `Why source{d}` (no changes)
- `Products` (new item)
  - `source{d} Community Edition` (links to https://landing-staging.srcd.run/features)
  - `source{d} Enterprise` (link to https://landing-staging.srcd.run/enterprise)
- ~`Solutions`~ (will contain 4 new pages, but not yet ready, better in a PR with that 4 new pages)
- `Community` (reordered, but same content)
- `Resources` (reordered, and new item inside)
  - add `newsletter`
- `Company` (no changes)

@vcoisne Need clarifications for:
- what's the url for `newsletter`? maybe http://go.sourced.tech/newsletter?
- since current `features` will be renamed as `source{d} Community Edition`, should its current link (https://landing-staging.srcd.run/features) be changed? If so, that old link is already being used in the _Welcome_ dashboard of **source{d} CE**, being PRed by https://github.com/src-d/sourced-ui/pull/164/files, so I think @marnovo should be noticed about the final URL for that page.

**Comment by vcoisne** [comment]
Thank you for the great summary of the requirements @dpordomingo 

Yes that's the right URL for the newsletter

Yes the URL should be changed from https://landing-staging.srcd.run/features to https://landing-staging.srcd.run/community-edition and updated in the product cc @marnovo @znegrin

**Comment by vcoisne** [comment]
URL should be updated as follow:

https://sourced.tech/features/ -> https://sourced.tech/products/community-edition
https://sourced.tech/enterprise/ -> https://sourced.tech/product/enterprise-edition

URL for the solutions pages:

https://sourced.tech/solutions/it-modernization-and-compliance
https://sourced.tech/solutions/devops-cloud-native-transformation
https://sourced.tech/solutions/engineering-effectiveness-efficiency
https://sourced.tech/solutions/talent-assesment-and-management

**Comment by znegrin** [comment]
I updated the header and footer with all the changes. Will change now the url on the proper Enterprise and Community edition pages

image

image

image

image

**Comment by dpordomingo** [comment]
already in staging https://landing-staging.srcd.run
should we merge @vcoisne ?

**Comment by znegrin** [comment]
I uploaded the changes requested on #424

**Pull request merged**